src/components/OutboxMessageListItem.vue

```diff
@@ -151,7 +151,7 @@ export default {
 				failed: false,
 				sendAt: (new Date().getTime() + UNDO_DELAY) / 1000,
 			}
-			await this.outboxStore.updateMessage({ message, id: message.id })
+			await this.outboxStore.sendMessage({ id: message.id, force: true })
 			try {
 				if (this.message.status !== STATUS_IMAP_SENT_MAILBOX_FAIL) {
 					await this.outboxStore.sendMessageWithUndo({ id: message.id })
```

Comment:
If this line is reached, the message would be resent (if the outbox still has it) or cause an error

Reply:
The *else* is even more problematic. Messages with status=STATUS_IMAP_SENT_MAILBOX_FAIL must not be sent again but just copied to the \Sent mailbox